go/common/docker/docker.go

```diff
@@ -210,7 +211,7 @@ func createNetwork(networkName string, cli *client.Client) error {
 func waitAndPullRemoteImage(image string, cli *client.Client) error {
 	// Pull the image from remote
 	fmt.Printf("Image %s not found locally. Pulling from remote...\n", image)
-	pullReader, err := cli.ImagePull(context.Background(), image, types.ImagePullOptions{})
+	pullReader, err := cli.ImagePull(context.Background(), image, image2.PullOptions{})
```

Comment:
That image2 auto-rename is a bit ugly, guess cos it clashed with that variable name. Maybe worth tweaking?